### DISCUSSION TITLE
Externship program

### DISCUSSION THREAD

**Discussion by reubenwenisch** [github/india#843] [GitHub Constellation 🇮🇳 2022 Q&A]
How do orgs/ startups register for externship program?

**Comment by arkodyuti** [comment]
Hi @reubenwenisch 👋🏻 currently we are not accepting applications for organisations to join the GitHub Externship program. However, you can drop an email at: india-externships@github.com and we will keep you posted once we open applications. Thanks!

**Comment by FRTMentor** [comment]
Please reach out our support team.
support@futurereadytalent.in

**Comment by arkodyuti** [comment]
Hi @reubenwenisch 👋🏻 we just announced the newly rebranded global GitHub Octernships program.🥳 If you're interested to participate as a partner organisation, sign up [here](https://aka.ms/OcternshipPartnerApplication). Find more about the program on our [website](https://education.github.com/students/octernships).
